e2e/src/usecases/SecureSend.js

```diff
@@ -1,92 +1,175 @@
 import {
-  SAMPLE_BACKUP_KEY_VERIFIED,
+  SAMPLE_BACKUP_KEY_SINGLE_ADDRESS_VERIFIED,
+  SAMPLE_PRIVATE_KEY,
   VERIFIED_PHONE_NUMBER,
+  SAMPLE_WALLET_ADDRESS_SINGLE_ADDRESS_VERIFIED,
   SAMPLE_WALLET_ADDRESS_VERIFIED_2,
 } from '../utils/consts'
 import { launchApp } from '../utils/retries'
 import {
   addComment,
   enterPinUiIfNecessary,
+  fundWallet,
   inputNumberKeypad,
   scrollIntoView,
   quickOnboarding,
+  waitForElementId,
+  waitForElementByIdAndTap,
 } from '../utils/utils'
 const faker = require('@faker-js/faker')
 
 const AMOUNT_TO_SEND = '0.01'
 
 export default SecureSend = () => {
-  describe.each([{ web3Library: 'contract-kit' }, { web3Library: 'viem' }])(
-    'Secure send flow with phone number lookup (with $web3Library)',
-    ({ web3Library }) => {
-      beforeAll(async () => {
-        // uninstall the app to remove secure send mapping
-        await device.uninstallApp()
-        await device.installApp()
-        await launchApp({
-          newInstance: true,
-          permissions: { notifications: 'YES', contacts: 'YES' },
-          launchArgs: {
-            statsigGateOverrides: `use_new_send_flow=false,use_viem_for_send=${
-              web3Library === 'viem'
-            }`,
-          },
-        })
-        await quickOnboarding(SAMPLE_BACKUP_KEY_VERIFIED)
+  describe('Secure send flow with phone number lookup (old flow)', () => {
+    beforeAll(async () => {
+      // uninstall the app to remove secure send mapping
+      await device.uninstallApp()
+      await device.installApp()
+      // fund wallet for send
+      await fundWallet(
+        SAMPLE_PRIVATE_KEY,
+        SAMPLE_WALLET_ADDRESS_SINGLE_ADDRESS_VERIFIED,
+        'cUSD',
+        `${AMOUNT_TO_SEND * 2}`
```

Comment:
Is `AMOUNT_TO_SEND * 2` enough to cover the send amount and fee's? Funding the wallet each test run might be error prone and instead we should just add this wallet to`./e2e/scripts/fund-e2e-accounts.ts`; however, that script is currently failing so `AMOUNT_TO_SEND * 2.2` might be enough for this PR.